test/groovy/util/JenkinsShellCallRule.groovy

```diff
@@ -24,12 +24,12 @@ class JenkinsShellCallRule implements TestRule {
         }
 
         @Override
-        public int hashCode() {
+        int hashCode() {
             return type.hashCode() * script.hashCode()
         }
 
         @Override
-        public boolean equals(Object obj) {
+        boolean equals(Object obj) {
```

Comment:
[minor] I wounder why this compiles. When overwriting superclass methods it is normally not possible to reduce visibility. I feel better if we keep the `public` modifier.

Reply:
In groovy this is the default visibility and therefore redundant.